tests/acceptance/bootstrap/SearchContext.php

```diff
@@ -220,4 +218,84 @@ public function theSearchResultShouldContainEntriesWithHighlight(
 			);
 		}
 	}
+
+	/**
+	 * @When /^user "([^"]*)" searches for "([^"]*)" inside (folder|space) "([^"]*)"(?: in space "([^"]*)")? using the WebDAV API$/
+	 *
+	 * @param string $user
+	 * @param string $pattern
+	 * @param string $scope
+	 * @param string|null $scopeName
+	 * @param string|null $spaceName
+	 *
+	 * @return void
+	 */
+	public function userSearchesInsideFolderOrSpaceUsingWebDavAPI(
+		string    $user,
+		string    $pattern,
+		string    $scope,
+		string    $scopeName = null,
+		string    $spaceName = null
+	): void {
+		// NOTE: because indexing of newly uploaded files or directories with ocis is decoupled and occurs asynchronously
+		// short wait is necessary before searching
```

Comment:
```suggestion
		// NOTE: short wait is necessary before searching because indexing of newly uploaded files or directories with ocis is decoupled and occurs asynchronously
		//